data-model/src/main/scala/za/co/absa/enceladus/model/properties/PropertyDefinition.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.model.properties
+
+import java.time.ZonedDateTime
+
+import com.fasterxml.jackson.databind.node.{ArrayNode, ObjectNode}
+import za.co.absa.enceladus.model.menas.MenasReference
+import za.co.absa.enceladus.model.menas.audit.{AuditFieldName, AuditTrailChange, AuditTrailEntry, Auditable}
+import za.co.absa.enceladus.model.properties.essentiality.{Essentiality, Mandatory, Optional}
+import za.co.absa.enceladus.model.properties.propertyType.PropertyType
+import za.co.absa.enceladus.model.versionedModel.VersionedModel
+
+case class PropertyDefinition(name: String,
+                              version: Int = 1,
```

Comment:
Right, as far as I remember

Reply:
Exactly. There has been discussions around this, this seems to be the latest general decision. (could be redone if we wanted to in the future.)

Reply:
Yes, this is what was the latest decision I recall.

It could be redone so that the datasets hold definition versions for their property definitions, too. I have suggested this option a meeting we held.